platform/qt/src/format_number.cpp

```diff
@@ -0,0 +1,26 @@
+#include <mbgl/util/platform.hpp>
+
+#include <QLocale>
+#include <QString>
+
+namespace mbgl {
+namespace platform {
+
+std::string formatNumber(double number, const std::string& localeId, const std::string& currency,
+                         uint8_t minFractionDigits, uint8_t maxFractionDigits) {
+
+    QString formatted;
+    // Qt Locale::toString() API takes only one precision argument
+    (void)minFractionDigits;
```

Comment:
nit: you don't need to do this, you can just remove `minFractionDigits` from the function signature.